src/webgl/p5.RendererGL.js

```diff
@@ -26,10 +15,20 @@ var attributes = {
  * rendering (FBO).
  *
  */
-p5.RendererGL = function(elt, pInst, isMainCanvas) {
+p5.RendererGL = function(elt, pInst, isMainCanvas, attr) {
   p5.Renderer.call(this, elt, pInst, isMainCanvas);
+  this.attributes = {};
+  attr = attr || {};
+  this.attributes.alpha = attr.alpha === undefined ? true : attr.alpha;
+  this.attributes.depth = attr.depth === undefined ? true : attr.depth;
+  this.attributes.stencil = attr.stencil === undefined ? true : attr.stencil;
+  this.attributes.antialias = attr.antialias ===
```

Comment:
I agree but p5 has been strict line length rules for building and even if it is split at the `?` the last couple of attributes are still too long to build. I am still splitting at the `===` but I changed it so even the shorter attributes do this for consistency.